docs/code_ref/coordinates.rst

```diff
@@ -72,7 +72,7 @@ Accessing Coordinates
 
 Individual coordinates can be accessed via attributes on the SkyCoord object,
 but the names of the components of the coordinates for each frame differ. For a full
-description of all the properties of the frames see `sunpy.coordinates.frames`.
+description of all the properties of the frames see `~sunpy.coordinates.frames`.
```

Comment:
this reads better without the `~` to me.

Reply:
it links anything in single ticks.

the `~` changes the text for the link from 'sunpy.coordinates.frames' to 'frames', i.e. the thing after the last dot.

Reply:
Oh i had no idea. Ill change it back.